### ISSUE TITLE
Missing options to add description/alternative text

### ISSUE THREAD

**Issue by Semir1212** [humhub/humhub#5978]
Would improve accessibility and is currently missing for:

- Profile picture
- Profile banner
- Space icon
- Space banner

**Comment by luke-** [comment]
Currently, the following is provided as Alt Text for all profile images:
![image](https://user-images.githubusercontent.com/4736168/207697396-4b15baeb-fc85-4459-b451-c5fb5c0bf823.png)
Similar for Space Images.

No Alt text  is set for banner images. Here we should also add some Alt text..
